crates/rolldown/tests/rolldown/issues/4129/artifacts.snap

```diff
@@ -12,7 +12,7 @@ import assert from "node:assert";
 //#region lib.js
 var require_lib = __commonJS({ "lib.js"(exports, module) {
 	import.meta.hot = __rolldown_runtime__.createModuleHotContext("lib.js");
-	__rolldown_runtime__.registerModule("lib.js", module.exports);
+	__rolldown_runtime__.registerModule("lib.js", module.exports, { cjs: true });
```

Comment:
This still fails case like

```
module.exports = ...
```

We should probably move the `registerModule` call to the bottom.